### PR TITLE
Scenario: machine overview

### PR DESCRIPTION
**Pull request by manne**

### CODE REVIEW

File: src/web/Configurations/TapioCloud.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Aitgmbh.Tapio.Developerapp.Web.Configurations
+{
+    public class TapioCloud
```
Comment (SebastianRoether) [inline src/web/Configurations/TapioCloud.cs]: TapioCloudCredentials

File: src/web/Models/ScenarioAttribute.cs [hunk 2]
```diff
@@ -1,8 +1,13 @@
 using System;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Aitgmbh.Tapio.Developerapp.Web.Models
 {
-    [AttributeUsage(validOn: AttributeTargets.Class, AllowMultiple = false)]
+#pragma warning disable S1309 // Track uses of in-source issue suppressions
+    [SuppressMessage("Major Code Smell", "S3996:URI properties should not be strings", Justification = "It is just an dto")]
```
Comment (SebastianRoether) [inline src/web/Models/ScenarioAttribute.cs]: Why don't you just make it an URI?

File: src/web/Scenarios/MachineOverview/MachineOverviewService.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        public MachineOverviewService(HttpClient httpClient, ITokenProvider tokenProvider)
+        {
+            _httpClient = httpClient ?? throw new ArgumentNullException(nameof(httpClient));
+            _tokenProvider = tokenProvider ?? throw new ArgumentNullException(nameof(tokenProvider));
+        }
+
+        public async Task<SubscriptionOverview> GetSubscriptionAsync(CancellationToken cancellationToken)
+        {
+            var token = await _tokenProvider.ReceiveTokenAsync(cancellationToken);
+            var requestUri = new Uri(TapioGlobalDiscoSubscriptionOverview);
```
Comment (SebastianRoether) [inline src/web/Scenarios/MachineOverview/MachineOverviewService.cs]: Why not just use a readonly Uri in the first place?

File: src/web/Models/ScenarioAttribute.cs (hunk 2 above)
Comment (manne) [reply src/web/Models/ScenarioAttribute.cs]: It's a C# restriction <https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/attributes#attribute-parameter-types>

File: src/web/Configurations/TapioCloud.cs (hunk 1 above)
Comment (manne) [reply src/web/Configurations/TapioCloud.cs]: done

File: src/web/Scenarios/MachineOverview/MachineOverviewService.cs (hunk 3 above)
Comment (manne) [reply src/web/Scenarios/MachineOverview/MachineOverviewService.cs]: done

**Pull request merged**